Trimble.Modus.Components/Controls/Toast/TMToastContents.xaml.cs

```diff
@@ -146,7 +146,7 @@ private void PopupData(string message, string actionButtonText, Action action)
 
         var idiom = Device.Idiom;
         setWidth(idiom);
-        Message = GetWrappedLabelText(message, idiom);
+        Message = message;
     }
 
     private void setWidth(TargetIdiom idiom)
```

Comment:
![image](https://github.com/trimble-oss/modus-mobile-maui-components/assets/130461522/450aec93-67dc-42e4-b354-620de37c91e7)
Github suggest to use DeviceIdiom can you please try that

Reply:
Is Resloved in another PR #101